### ISSUE TITLE
New Crystal patch reminder permanently appearing

### ISSUE THREAD

**Issue by Gollum69** [queicherius/runelite-time-tracking-reminder#39]
Just had the update and the Crystal patch reminder has been added, thank you for adding another patch however this is now permanently appearing, I've been to the patch and I have nothing planted in there. I've hopped worlds and refreshed RuneLite it still appears, 

I've uploaded a screenshot just showing the empty patch but the reminder saying its ready for harvesting still appearing.
https://i.imgur.com/MGU7VyO.png

**Comment by queicherius** [comment]
The reminder is shown when the patch is waiting for you to do something (in this case, planting a seed). So this is indented in this case. You can toggle the patch on and off if you'd like to hide it while you don't have acorns for example.

**Comment by Gollum69** [comment]
thanks chad. i dont think anyone has an endless amount of crystal seeds so this will always be appearing for most people, ill have to turn it off in the meantime

**Comment by queicherius** [comment]
I'll keep it open if someone wants to make a PR to hide it when nothing is planted (for this one plot only)

**Comment by Gollum69** [comment]
> I'll keep it open if someone wants to make a PR to hide it when nothing is planted (for this one plot only)

yeah i think that would be a better way of operating this particular patch. as far as im aware its relatively rare and u definitely cant upkeep the seeds everytime u consume the patch

**Comment by queicherius** [comment]
Closed via #54
